internal/code-generator/assets/utils.go.tmpl

```diff
@@ -0,0 +1,84 @@
+package slinguist
```

Comment:
instead of having all the code in the template, we can have only the regexp and the code in a real go file